pycsw/core/config.py

```diff
@@ -134,6 +135,8 @@ def __init__(self, prefix='csw30'):
                 'pycsw:Type': 'type',
                 # geometry, specified in OGC WKT
                 'pycsw:BoundingBox': 'wkt_geometry',
+                'pycsw:VertExtentMin': 'vert_extent_min',
+                'pycsw:VertExtentMax': 'vert_extent_max',
```

Comment:
We need to validate these queryables (see other comments in this review).

Reply:
@tomkralidis I have colleagues who work in ocean biosciences for whom this is a fundamental requirement. For myself, I require it to represent underground geology/geophysics datasets.